app/api/edlinker/user.rb

```diff
@@ -1,4 +1,22 @@
 class Edlinker::User < Grape::API
+  helpers Edlinker::Helpers::Auth
+  helpers Edlinker::Helpers::User
+
+    desc 'add students by teacher'
+    params { use :person_params }
+    post ':add_student' do
+      validate_teacher
+      user = User.new(email: params[:email], phone_number: params[:phone_number], first_name: params[:first_name],
+                      last_name: params[:last_name], patronymic: params[:patronymic])
+      user.password = params[:password]
+      user.add_role(:student)
+      user.save!
+      error!('Invalid username or password') unless user.present?
```

Comment:
it's better to show actual errors

error!(user.errors.full_messages) unless user.save